regression/config.sh

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+#
+# NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+# property and proprietary rights in and to this material, related
+# documentation and any modifications thereto. Any use, reproduction,
+# disclosure or distribution of this material and related documentation
+# without an express license agreement from NVIDIA CORPORATION or
+# its affiliates is strictly prohibited.
+
+THIS_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
+
+CUOPT_SCRIPTS_DIR=$THIS_DIR
+OUTPUT_DIR=$SCRATCH_DIR/benchmark_runs/
+
+ACCOUNT=datascience_rapids_testing
+PARTITION="batch"
+GPUS_PER_NODE=8
+
+# Path to the squashs file containing the container image
+IMAGE="nvidia/cuopt:25.10.0a-cuda12.9-py3.12"
```

Comment:
Lets try to get branch name from git tag or something else so we don't need to change this in each release.